subscribers/models.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from django.db import models
+
+# Create your models here.
+
+
+class subscribers(models.Model):
+    user = models.ForeignKey(User)
+    zip_code = models.CharField(max_length=10)
```

Comment:
I was wondering if it would make sense to include a sort of 'subscriber type/notification type/distance.' For example, someone might want to subscribe to notifications for their only their zip, their county, their state, some distance, maybe a researcher wants all notifications. Not sure if we'd want to build that in as it could cost us money to send for everything but some sort of concept of 'notification granularity' might be useful.

Reply:
For the subscriptions app we want create a zipcode table, which will include all of the U.S. zipcodes. 

We will will update the subscribers table to make the zip_code field a manytomany database field referencing the zipcode table. Also, we want an ip address field within subscribers to collect the ip address the user accessing the data.

Reply:
This has been updated (rebased commit and previous into one).